tests/performance/prewhere_with_row_level_filter.xml

```diff
@@ -1,16 +1,15 @@
 <test>
     <create_query>DROP TABLE IF EXISTS test_prl;</create_query>
     <create_query>CREATE TABLE test_prl (n UInt64) ENGINE MergeTree ORDER BY n;</create_query>
-    <!--<create_query>GRANT CREATE ROW POLICY ON *.* TO CURRENT_USER</create_query>-->
     <create_query>CREATE ROW POLICY OR REPLACE test_prl_policy ON test_prl AS PERMISSIVE FOR SELECT USING n % 7 TO ALL;</create_query>
 
-    <fill_query>INSERT INTO test_prl SELECT number FROM numbers(500000000);</fill_query>
+    <fill_query>INSERT INTO test_prl SELECT number FROM numbers(100000000);</fill_query>
```

Comment:
Maybe we need to enable parallel insert select

Reply:
Change to something like this, you mean?
```sql
INSERT INTO test_prl SELECT number FROM numbers(100000000) SETTINGS max_insert_threads = 10;
```

Reply:
yep

Reply:
Pushed the change.